modules/odc-creating-projects-using-developer-perspective.adoc

```diff
@@ -19,12 +19,16 @@ Projects starting with `openshift-` and `kube-` host cluster components that run
 .Procedure
 You can create a project using the *Developer* perspective, as follows:
 
-. In the *Add* view, click the *Project* drop-down menu to see a list of all available projects. Select *Create Project*.
+. Click the *Project* drop-down menu to see a list of all available projects. Select *Create Project*.
 +
 image::odc_create_project.png[Create Project]
 
-. In the *Create Project* dialog box, enter a unique name for the *Name* field. For example, enter `myproject` as the name of the project in the *Name* field.
-. Optional: Add the *Display Name* and *Description* details for the Project.
+. In the *Create Project* dialog box, enter a unique name for the *Name* field. For example, enter `myproject` as the name of the project.
+. Optional: Add the *Display Name* and *Description* details for the project.
 . Click *Create*.
-. Navigate to the *Advanced → Project Details* page to see the dashboard for your project.
-. In the *Project* drop-down menu at the top of the screen, select *all projects* to list all of the projects in your cluster. If you have adequate permissions for a project, you can use the  *Options* menu {kebab} to edit or delete the project.
+. Use the left navigation panel to navigate to the *Project* view and see the dashboard for your project.
+. Optional:
++
+* Use the *Project* drop-down menu at the top of the screen and select *all projects* to list all of the projects in your cluster.
+* Use the *Details* tab to see the project details.
+* If you have adequate permissions for a project, you can use the *Project Access* tab to provide or revoke; admin, edit, and view access to the project.
```

Comment:
s/to provide or revoke; admin, edit, and view/to provide or revoke, administer, edit, and view/

Reply:
Changing this to: 'to provide or revoke: administrator, edit, and view access to the project.'
The user can provide and revoke, all of these accesses. Using a comma, IMHO changes that meaning.